app/models/container_node_performance.rb

```diff
@@ -2,4 +2,8 @@ class ContainerNodePerformance < MetricRollup
   default_scope { where("resource_type = 'ContainerNode' and resource_id IS NOT NULL") }
 
   belongs_to :container_node, :foreign_key => :resource_id, :class_name => ContainerNode.name
+
+  def self.display_name(number = 1)
+    n_('Performance - Container Node', 'Performance - Container Nodes', number)
```

Comment:
Should these be "Container Node Performance" and "Container Node Performances"?  I think they are performance records for a single node.